functions/src/plugins/size.ts

```diff
@@ -110,50 +146,47 @@ export class SizeTask extends Task {
    * @param context Must be from a "Status" github event
    * @param artifacts
    */
-  async upsertNewArtifacts(context: Context, artifacts: BuildArtifact[]): Promise<void> {
-    this.logDebug(`[size] Storing artifacts for: ${context.payload.commit.sha}, on branches [${context.payload.branches.map((b: any) => b.commit.url).join(', ')}]`);
-
-    // eg: aio/gzip7/inline
-    // eg: ivy/gzip7/inline
-    // projects within this repo
-    const projects = new Set(artifacts.map(a => a.projectName));
-
-    const updates: any = {};
-
-    for(const project of projects) {
-      for(const branch of context.payload.branches) {
-        const artifactsOutput = {
-          change: 'application',
-          message: context.payload.commit.commit.message,
-          timestamp: new Date().getTime(),
-        };
-
-        // only use the artifacts from this project
-        artifacts.filter(a => a.projectName === project)
-          .forEach(a => {
-            // hold a ref to where we are in our tree walk
-            let lastNestedItemRef: any | number = artifactsOutput;
-            // first item is the project name which we've used already
-            a.contextPath.forEach((path, i) => {
-              const encodedPath = firebasePathEncode(path);
-              // last item so assign it the bytes size
-              if(i === a.contextPath.length - 1) {
-                lastNestedItemRef[encodedPath] = a.sizeBytes;
-                return;
-              }
-              if(!lastNestedItemRef[encodedPath]) {
-                lastNestedItemRef[encodedPath] = {};
-              }
-
-              lastNestedItemRef = lastNestedItemRef[encodedPath];
+  async upsertNewArtifacts(
+    context: Context,
+    artifacts: BuildArtifact[]
+  ): Promise<void> {
```

Comment:
Can you put the function definition on one line instead of 4 ?

Reply:
Done.